install/kubernetes/tetragon/templates/_container_tetragon.tpl

```diff
@@ -70,14 +70,9 @@
 {{- else if .Values.tetragon.grpc.enabled }}
   livenessProbe:
      timeoutSeconds: 60
-     exec:
-       command:
-       - tetra
-       - status
-       - --server-address
-       - {{ .Values.tetragon.grpc.address }}
-       - --retries
-       - "5"
+     httpGet:
+       path: /liveness
+       port: 6789
```

Comment:
This is a bit strange. We enable the `httpGet` probe under `grpc.enabled`.

I think it would make more sense to an option to select the http probe as was done in https://github.com/cilium/tetragon/pull/2469.

Could we also split this into multiple patches? One patch for the agent and one patch for the helm changes?

Reply:
Thanks, I have updated the PR based on that comment.